apps/api/src/server/repositories/document.repository.js

```diff
@@ -22,7 +22,29 @@ export const upsert = (document) => {
 export const getById = (documentGuid) => {
 	return databaseConnector.document.findUnique({
 		where: {
-			guid: documentGuid
+			guid_isDeleted: {
```

Comment:
Is this a typo or is this some weird syntax for prisma?

Reply:
fun prisma syntax